### DISCUSSION TITLE
Web UI for maintaining list(s) of applications for Chezmoi to install

### DISCUSSION THREAD

**Discussion by johan-weitner** [twpayne/chezmoi#3901] [Show and tell]
Hi. 

I built a little UI for managing the list(s) of applications that Chezmoi installs. So, @twpayne, let me know if you want your product name (and logo) taken off of it. And any input is of course welcome.

Repo: [https://github.com/johan-weitner/chezmoi-ui](https://github.com/johan-weitner/chezmoi-ui)

Regards,
Johan Weitner

**Comment by twpayne** [comment]
Thanks for this! I've added chezmoi-ui to the related software page in ca0d13356da87e4dcb3dc508dc830cf347fbdaf1.

**Reply by johan-weitner** [reply]
Thanks!

**Comment by Lockszmith-GH** [comment]
Interesting! Could you please share some screenshots?

**Reply by johan-weitner** [reply]
If you scroll down the README you should find a few screenshots.

**Reply by Lockszmith-GH** [reply]
Ah, I see in the source you have UI examples, but they are not rendering on my end

The markdown
![image](https://github.com/user-attachments/assets/9bb4cd7e-290d-4a2b-8fab-dc71cbfb1ede)

What I see
![image](https://github.com/user-attachments/assets/7eeaac52-47a8-4679-ba5d-656d14e28db9)

But this might be on my end - Maybe something blocked by my employer's gateway - I'll check from home later.

**Reply by Lockszmith-GH** [reply]
I cloned the repo to my laptop and I see everything, thanks.

**Reply by johan-weitner** [reply]
OK, cool. The screenshots are located in /assets, in case the README doesn't work. Appreciate the interest.
